spec/services/merge_requests/create_service_spec.rb

```diff
@@ -6,20 +6,29 @@
 
   describe :execute do
     context "valid params" do
+      let(:opts) do
+        {
+            title: 'Awesome merge_request',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.